### DISCUSSION TITLE
Single image sharing

### DISCUSSION THREAD

**Discussion by pablo03v** [ente-io/ente#859] [Enhancements]
**Describe the feature**
Instead of sharing the image as image I'd like to send an image via link sometimes. Would come in handy, especially, when the communication channel is propriatary or not encrypted.

**Workaround**
Creating an album with a single photo and share the album link.

**Comment by windmueller** [comment]
This should not be limited to sharing with links. Sometimes I would like to share a single image with family members.

**Comment by taivlam** [comment]
This should be possible now using the "Share link" option on the mobile Ente Photos apps.
